contributingGuides/TESTING_APPLE_GOOGLE_SIGNIN.md

```diff
@@ -0,0 +1,128 @@
+# Testing Apple/Google sign-in
+
+Due to some technical constraints, Apple and Google sign-in may require additional configuration to be able to work in the development environment as expected. This document describes any additional steps for each platform. 
+
+## Apple
+
+### Web
+
+The Sign in with Apple process will break after the user signs in if the pop-up process is not started from a page at an HTTPS domain registered with Apple. We can make a new configuration with your custom HTTPS domain, but then the Apple configuration won't match that of Expensify's backend.
```

Comment:
Minor change here, since contributors will view this documentation, it could create confusion if it implies that Expensify will make the configuration for them.

Reply:
Good shout, addressed this